api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>	
 
```

Comment:
I thought the ticket was not about the 1.5 config. So i would not expect any changes in this file.